metatron-app/metatron-discovery
# F#805 add file upload location

### Description

If the dataset has file type, user should choose a destination

**Related Issue** : 

[#805](https://github.com/metatron-app/metatron-discovery/issues/805)

### How Has This Been Tested?

1. create a file dataset
2. upload a file and select a upload location
3. if the dataset is created, check the property 'fileType'

### Types of changes



### Checklist:



### Additional Context
